### DISCUSSION TITLE
Tips

### DISCUSSION THREAD

**Discussion by tomishima2904** [tomishima2904/es_manager#2] [General]
いろんな情報を共有しよう

**Comment by tomishima2904** [comment]
sourcetreeを使ったgit操作をする際、リモートリポジトリの登録には **http** を使う！じゃないとエラー出る
[参考文献](https://kantaro-cgi.com/blog/git/bitbucket-permission-error.html)

**Comment by tomishima2904** [comment]
[.envファイルを使ったdocker-compose.ymlの環境変数設計](https://blog.cloud-acct.com/posts/u-env-docker-compose/)

**Comment by tomishima2904** [comment]
VScode拡張機能

**Reply by tomishima2904** [reply]
名前: Spring Boot Extension Pack
ID: vmware.vscode-boot-dev-pack
説明: A collection of extensions for developing Spring Boot applications
バージョン: 0.2.1
パブリッシャー: VMware
VS Marketplace リンク: https://marketplace.visualstudio.com/items?itemName=vmware.vscode-boot-dev-pack

**Reply by tomishima2904** [reply]
名前: Trailing Spaces
ID: shardulm94.trailing-spaces
説明: Highlight trailing spaces and delete them in a flash!
バージョン: 0.4.1
パブリッシャー: Shardul Mahadik
VS Marketplace リンク: https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces

**Reply by mctomoney** [reply]
done

**Comment by tomishima2904** [comment]
よく使うコマンド

**Reply by tomishima2904** [reply]
# DB

## コンテナ外（`$`はプロンプト）

MySQLのコンテナに入る
```
$ docker compose exec db bash
```

## コンテナ内（`bash-4.4#`はプロンプト）

MySQLにログイン（ユーザー名指定して、パスワード利用）
```
bash-4.4# mysql -u <ユーザー名> -p
```

## MySQL上（`mysql>`はプロンプト）

使用するDBを選択。忘れがちなので注意。

```
mysql> use <DB名>;
```

データベースにあるテーブル一覧を取得

```
mysql> show tables;
```

テーブルの内容を取得

```
mysql> select * from <テーブル名>;
```

**Comment by tomishima2904** [comment]
[gitのコミットメッセージの書き方](https://zenn.dev/itosho/articles/git-commit-message-2023)
